documentation/library_guide/index.rst

```diff
@@ -10,24 +10,36 @@ For general information, refer to the `oneDPL GitHub* repository
 
 .. toctree::
    :maxdepth: 2
+   :titlesonly:
    :caption: Get Started
 
    introduction/release_notes.rst
    introduction/onedpl_gsg.rst
 
 .. toctree::
    :maxdepth: 2
-   :caption: Library Guide
+   :titlesonly:
+   :includehidden:
+   :caption: Library Core Guide
```

Comment:
For both "Library Core Guide" and "Technology Preview Guide" sections, I would remove "Guide". I think of "Guide" as a document and not a subsection of a larger "Guide". Otherwise I think it looks good, and this suggestion is just a preference.

Reply:
Changed to "Core Functionality" and "Technology Preview"